backend/app.py

```diff
@@ -542,6 +542,30 @@ def get_top_fellows():
             "message": f"Error: {e}"
         })
 
+
+@app.route("/get_total_registered_fellows")
+@jwt_required()
+def get_total_fellows():
+    """Retrieve the number of fellows in the app.
+
+    Returns:
+        json: json payload containing requested information
+    """
+    try:
+        num_fellows = User.query.count()
```

Comment:
We need the count of users that are not admin.
maybe get a count with users with admin role and subtract that.

Reply:
You can do User.query.filter( User.role != 'admin').count()